From 4e35f3b7621f6ff22f1f2b76d4a95ff32fc401b3 Mon Sep 17 00:00:00 2001
From: Kartatz <105828205+Kartatz@users.noreply.github.com>
Date: Fri, 17 May 2024 16:43:54 +0200
Subject: [PATCH] Add libversions

This is dynamically generated by the Makefile in OpenBSD ports, but since we are not using it, let's define those values manually
---
 libversions | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 libversions

diff --git a/libversions b/libversions
new file mode 100644
index 0000000..f3e875d
--- /dev/null
+++ b/libversions
@@ -0,0 +1,12 @@
+LIBestdc++_LTVERSION = -version-info 20:0
+LIBgfortran_LTVERSION = -version-info 9:0
+LIBobjc_LTVERSION = -version-info 9:0
+LIBlto_plugin_LTVERSION = -version-info 6:0
+LIBitm_LTVERSION = -version-info 5:0
+LIBatomic_LTVERSION = -version-info 4:0
+LIBquadmath_LTVERSION = -version-info 4:0
+LIBcc1_LTVERSION = -version-info 2:0
+LIBcc1plugin_LTVERSION = -version-info 2:0
+LIBcp1plugin_LTVERSION = -version-info 1:0
+LIBgphobos_LTVERSION = -version-info 0:0
+LIBgdruntime_LTVERSION = -version-info 0:0
-- 
2.36.6

